digdag-docs/src/operators/for_each.md

```diff
@@ -38,16 +38,26 @@
   for_each>: {i: '[1, 2, 3]'}
   ```
 
-* **\_parallel**: BOOLEAN
+* **\_parallel**: BOOLEAN | OBJECT
 
   Runs the repeating tasks in parallel.
+  If ``_parallel: {limit: N}`` (N is an integer: 1, 2, 3, …) parameter is set,
+  the number of loops running in parallel is limited to N.
```

Comment:
`the number of tasks running in parallel` is a bit clearer?

Reply:
Fixed: ad5ea45